### PR TITLE
Update Collect setup for projects release

### PR DESCRIPTION
**Pull request by seadowg**
Work towards #1365

#### What is included in this PR?

Update ODK "Getting Started" and Collect's "Connecting to a Server" for Projects and add the beginning of a "Managing Projects" page. This doesn't address everything we need to update and there is still more to document on Projects themselves, but I think it's a good start and makes sure new users aren't looking at out of date setup instructions.

### CODE REVIEW

File: src/collect-connect.rst [hunk 1]
```diff
@@ -3,26 +3,22 @@ Connecting to a Server
 
 ODK Collect is used to fill forms with participants. Filled forms then need to be sent to a central location for review and analysis. Generally, organizations do this by configuring Collect to send forms to a server. For those working in environments without any internet connectivity, there are :ref:`other options <other-collect-server-options>`.
```
Comment (yanokwa) [inline src/collect-connect.rst:4]: Drop the last two sentences (Generally...For those working).

**Review by yanokwa — CHANGES_REQUESTED** [review]
I think this is a good first pass because it's better to have some docs than no docs. @lognaturel will want to review this too, but I wanted to sneak in my feedback before she did.

We'll merge after you respond to my feedback and she can come over the top with her own PR if she'd like.

**Comment by seadowg** [comment]
@yanokwa agreed. Much more to do but it feels like this at least catches new users who might be very confused that the setup steps are out of sync. I'll go through and make the changes you suggested/fix the mistakes you caught.

File: src/collect-projects.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.. _collect-projects:
+
+Managing Projects
+=================
+
+.. versionadded:: v2021.2
+
+Starting with Collect v2021.2, Collect can be setup with multiple Projects. Each Project in Collect has its own set of :doc:`Forms <collect-forms>` and Settings.
+
+Each Project is identified by a name, icon and color. These are then shown in at the top of the :ref:`Main Menu <main-menu>`:
```
[the remark this answers is not in the document]
Comment (seadowg) [reply src/collect-projects.rst:10]: The name is on the left and the icon is on the right, so I just went with "top" here.

File: src/collect-connect.rst [hunk 3]
```diff
@@ -63,4 +46,4 @@ Other options
 
   collect-connect-google
   Transfering blank and completed forms directly with adb <collect-adb>
-  Using ODK Briefcase  <briefcase-using>
\ No newline at end of file
+  Using ODK Briefcase  <briefcase-using>
```
[the remark this answers is not in the document]
Comment (seadowg) [reply src/collect-connect.rst:49]: Oh, I'm not sure about ADB. We're still thinking of it as a pretty normal workflow (we've got analytics in to track this finally as well) and definitely the best solution for offline work. Can we just leave these in for this PR and come back to that kind of change?

File: src/collect-connect.rst (hunk 1 above)
Comment (seadowg) [reply src/collect-connect.rst:4]: Yeah, like my comment above, these feels important to maintain direction for folks needing to do offline set up.

File: src/collect-connect.rst [hunk 4]
```diff
@@ -3,26 +3,22 @@ Connecting to a Server
...
-  
-Once you are done trying out Collect, you will need a plan for managing forms and data submissions. We recommend using :ref:`ODK Central <central-intro>` and configuring Collect by QR code. :doc:`Central <central-intro>` provides user and project management features as well as tools for viewing and exporting data. For complex data collection projects, it is usually the right choice. Organizations can choose to use their own infrastructure and have total control over their server configuration. However, setting up and maintaining a server requires technical skills.
+We recommend using :ref:`ODK Central <central-intro>` as your server and configuring Collect by QR code. :doc:`Central <central-intro>` provides user and project management features as well as tools for viewing and exporting data. For complex data collection projects, it is usually the right choice. Organizations can choose to use their own infrastructure and have total control over their server configuration. However, setting up and maintaining a server requires technical skills.
```
[the remark this answers is not in the document]
Comment (seadowg) [reply src/collect-connect.rst:6]: Great shout.

File: src/collect-adb.rst [hunk 5]
```diff
@@ -206,7 +206,6 @@ The ODK Collect directory on your device is:
 
 * :file:`/sdcard/odk` if you are running an ODK Collect version less than v1.26.0 or have a file migration banner on the main screen
 * :file:`/sdcard/Android/data/org.odk.collect.android/files` if you have ODK Collect version v1.26.0+ and don't have a file migration banner on the main screen
+* If you have Collect 2021.2 or later then each :ref:`Project <collect-projects>` will have its own directory which can be found in :file:`/sdcard/Android/data/org.odk.collect.android/files/projects`. The Project directories will contain a blank file with the same name as the Project itself so you can identify it.
```
[the remark this answers is not in the document]
Comment (seadowg) [reply src/collect-adb.rst:209]: That looks better

**Comment by seadowg** [comment]
@yanokwa ok changes made. Feel free to just make tweaks if you want before merging!

**Pull request merged**